src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -730,42 +758,73 @@ public static partial class NativeLibrary
-        public static void SetDllImportResolver(System.Reflection.Assembly assembly, DllImportResolver resolver) { throw null; }
-        public static bool TryGetExport(IntPtr handle, string name, out IntPtr address) { throw null; }
+        public static System.IntPtr Load(string libraryName, System.Reflection.Assembly assembly, System.Nullable<System.Runtime.InteropServices.DllImportSearchPath> searchPath) { throw null; }
+        public static void SetDllImportResolver(System.Reflection.Assembly assembly, System.Runtime.InteropServices.DllImportResolver resolver) { }
+        public static bool TryGetExport(System.IntPtr handle, string name, out System.IntPtr address) { throw null; }
         public static bool TryLoad(string libraryPath, out System.IntPtr handle) { throw null; }
-        public static bool TryLoad(string libraryName, System.Reflection.Assembly assembly, DllImportSearchPath? searchPath, out System.IntPtr handle) { throw null; }
+        public static bool TryLoad(string libraryName, System.Reflection.Assembly assembly, System.Nullable<System.Runtime.InteropServices.DllImportSearchPath> searchPath, out System.IntPtr handle) { throw null; }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(2048), Inherited = false)]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Parameter, Inherited=false)]
     public sealed partial class OptionalAttribute : System.Attribute
     {
         public OptionalAttribute() { }
     }
-    [System.AttributeUsageAttribute((System.AttributeTargets)(64), Inherited = false)]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Method, Inherited=false)]
     public sealed partial class PreserveSigAttribute : System.Attribute
     {
         public PreserveSigAttribute() { }
     }
-    [AttributeUsage(AttributeTargets.Assembly, Inherited = false, AllowMultiple = true)]
-    public sealed class PrimaryInteropAssemblyAttribute : Attribute
+    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly, Inherited=false, AllowMultiple=true)]
+    public sealed partial class PrimaryInteropAssemblyAttribute : System.Attribute
     {
         public PrimaryInteropAssemblyAttribute(int major, int minor) { }
-        public int MajorVersion { get; }
-        public int MinorVersion { get; }
+        public int MajorVersion { get { throw null; } }
+        public int MinorVersion { get { throw null; } }
     }
-    [AttributeUsage(AttributeTargets.Class, Inherited = false)]
-    public sealed class ProgIdAttribute : Attribute
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class, Inherited=false)]
+    public sealed partial class ProgIdAttribute : System.Attribute
     {
         public ProgIdAttribute(string progId) { }
-        public string Value { get; }
+        public string Value { get { throw null; } }
+    }
+    [System.FlagsAttribute]
+    public enum RegistrationClassContext
```

Comment:
Where is this coming from?

Reply:
It exists in netfx but doesn't seem to be there in netstandard: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.registrationclasscontext?view=netframework-4.7.2&viewFallbackFrom=netcore-3.0

https://github.com/dotnet/standard/blob/205332796a9a0f869ff698402c3496f14cc71f50/docs/comparisons/netstandard2.0_vs_net461/System.Runtime.InteropServices.md
https://apisof.net/catalog/System.Runtime.InteropServices.RegistrationClassContext

It was added here: https://github.com/dotnet/corefx/pull/12873 to fix https://github.com/dotnet/corefx/issues/12162

cc @alexperovich, do you recall if there was a reason for not adding this to the ref?